tox.ini

```diff
@@ -1,5 +1,6 @@
 [tox]
-envlist = py35,py36,py27,lint
+envlist = {py35,py36,py38,py27}{,-win},lint
```

Comment:
{,-win} is also added in #1537. You do not need to add it here.

Reply:
Updated